### PR TITLE
Added base form validation logic

### LINKED ISSUE #75: Field set validation

### PR DESCRIPTION
**Pull request by IgorZiegler**
- Form should validate fields until some field will be invalid or all field will be valid
 - Fieldset check all field

fixes #75

### CODE REVIEW

**Comment by IgorZiegler** [comment]
fixes #75

**Comment by 102** [comment]
>fixes #75

This should be in the pr body

File: frontend/blocks/fieldset/fieldset.js [hunk 1]
```diff
@@ -53,6 +53,12 @@ export default class Fieldset extends Container {
     return row;
   }
 
+  validate() {
+    return this.items.every(block => {
+      return block.validate();
```
Comment (102) [inline frontend/blocks/fieldset/fieldset.js]:
```js
block => block.validate();
```

File: frontend/blocks/form/form.js [hunk 2]
```diff
@@ -7,6 +7,12 @@ export default class Form extends Container {
     return template;
   }
 
+  validate() {
+    return this.items.some(block => {
+      return !block.validate();
```
Comment (102) [inline frontend/blocks/form/form.js]: Why form and fieldset validations are opposite?

Comment (102) [inline frontend/blocks/form/form.js]: And this form should be used here as well: `block => block.validate();`

**Pull request merged**